pajbot/modules/pnsl.py

```diff
@@ -73,6 +73,7 @@ def run_pnsl(self, bot, source, message, event, args):
             return False
 
         guid = message.replace("https://bot.tetyys.com/BotList/", "")
```

Comment:
huh? shouldnt this line be removed

Reply:
If anything, this line should stay but the one under it (76) should go/be corrected. 
Being an /api link means it will be formatted like so: `https://bot.tetyys.com/api/v1/BotList/{guid}?Token={token}` - therefore not properly trimming the `Token` param. Unless I'm missing something here?

Reply:
![image](https://user-images.githubusercontent.com/12804673/90501170-44db0a00-e18f-11ea-9659-3f86260f3823.png)
![image](https://user-images.githubusercontent.com/12804673/90501241-689e5000-e18f-11ea-9baa-e4cd47117d4b.png)
![image](https://user-images.githubusercontent.com/12804673/90501253-718f2180-e18f-11ea-88a5-ec607c803678.png)

Reply:
I mean like there's two assignments to `guid=`, the first one will always be overridden by the second one, no?

Reply:
Valid point in that case. I thought something like this wouldn't matter in Python (which I have little experience with), but I stand corrected.

Reply:
My understanding was that it works like a variable in javascript